build-system/tasks/visual-diff/index.js

```diff
@@ -131,10 +135,20 @@ async function launchPercyAgent() {
   }
 
   const env = argv.percy_agent_debug ? {LOG_LEVEL: 'debug'} : {};
-  percyAgentProcess_ = execScriptAsync('npx percy start', {
-    cwd: __dirname,
-    env: Object.assign(env, process.env),
-    stdio: ['ignore', process.stdout, process.stderr],
+  percyAgentProcess_ = execScriptAsync(
+    `npx percy start --port ${PERCY_AGENT_PORT}`,
+    {
+      cwd: __dirname,
+      env: Object.assign(env, process.env),
+      stdio: ['ignore', process.stdout, process.stderr],
+    }
+  );
+  await waitUntilUsed(
+    PERCY_AGENT_PORT,
+    PERCY_AGENT_RETRY_MS,
+    PERCY_AGENT_TIMEOUT_MS
+  ).then(() => {
+    log('info', 'Percy agent is reachable on port', PERCY_AGENT_PORT);
   });
```

Comment:
`await foo().then()` seems to be mixing `async/await` syntax with classic promise syntax. 

Maybe rewrite as follows?

```js
  await waitUntilUsed(
    PERCY_AGENT_PORT,
    PERCY_AGENT_RETRY_MS,
    PERCY_AGENT_TIMEOUT_MS
  );
  log('info', 'Percy agent is reachable on port', PERCY_AGENT_PORT);
```

Reply:
Done